routes/web.php

```diff
@@ -24,6 +24,7 @@
 Route::view('/team','theme::team')->name('team');
 Route::view('/contact','theme::contact')->name('contact');
 Route::view('/about-us','theme::about_us')->name('about-us');
+Route::view('/integrations','theme::integrations')->name('integrations');
```

Comment:
This works too, but would you create a simple controller serving this view, along with the SEO information.